notebooks/start-a-business-checker/users_visiting_two_start_a_business_content_items.sql

```diff
@@ -0,0 +1,13 @@
+
+select count(distinct Sessionid) as Sessions, CountofPages
+from (
+       select sessionid, count(distinct pagepath) as CountOfPages
```

Comment:
```suggestion
       select sessionid, count(distinct pagepath) as CountOfPages
```

The name of this variable is misleading. It is actually the `countOfDistinctPages` visited, I'd rename it.